config.go

```diff
@@ -57,6 +57,10 @@ type Config struct {
 	// This should be used only for testing.
 	InsecureSkipVerify bool
 
+	// InsecureHashes allows the use of hashing algorithms for which
+	// insecure() call returns true.
```

Comment:
`insecure()` is unexposed and not shown in godoc.
Would be better to be explained without unexposed function.